napari/layers/labels/labels.py

```diff
@@ -135,11 +135,21 @@ def __init__(
         blending='translucent',
         visible=True,
         multiscale=None,
+        color_dict=None,
     ):
 
         self._seed = seed
+        self._background_label = 0
         self._num_colors = num_colors
-        colormap = ('random', colormaps.label_colormap(self.num_colors))
+        self._random_colormap = (
+            'random',
+            colormaps.label_colormap(self.num_colors),
+        )
+        self._color_mode = LabelColorMode.AUTO
+        self._color_dict = {
```

Comment:
Do we actually need this initialization here if this get's overwritten in line `203`, `self.color_dict = color_dict`? If so can we just initialize as `{}` instead? I don't love having the `'transparent'` and `'black'` sneak in there because they use the string representation instead of the RBGA representation

Reply:
I managed to decouple the color dict and color mode, and removed the init from here